frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -31,9 +33,24 @@ const PipelinesOverview: React.FC<PipelinesOverviewProps> = ({
   const {
     metadata: { name, namespace },
   } = pipeline;
+  const [showAlert, setShowAlert] = React.useState(isPipelineNotStarted(name));
+
+  React.useEffect(() => {
+    setShowAlert(isPipelineNotStarted(name));
+  }, [name]);
```

Comment:
I think we want to just live read it on each render... without the hooks. This way wouldn't need to worry so much about unsetting it when the user clicks the 'x'.

```suggestion
  const showAlert = isPipelineNotStarted(name);
```

Reply:
removed UseEffect but still using local state as the re render is not quick enough to remove the Alert.

Reply:
Ah I see... my bad. 🤔